### PR TITLE
End to end part 3

### PR DESCRIPTION
**Pull request by Dave-infuseai**

### CODE REVIEW

File: docs/primehub-end-to-end-tutorial-3.md [hunk 1]
```diff
@@ -10,67 +10,119 @@ description: Using PrimeHub from Training to Serving the Model
...
 ![](assets/tutorial_models_managed.png)
 
 ## Customize Model Server Image
 
-Now, we have registered current best model in `Model Management`. To deploy the managed model, we need to setup the pre-packaged model image to be runnable container environment.
+To deploy our registered model, we need to set up the container environment. For this step we’ll be customizing a pre-packaged model image to suit our needs. This will demonstrate how to modify, build, and deploy a custom image using PrimeHub Deployments. 
+
+### Requirements
+
+To follow the instructions in this section you should have a the following:
```
Comment (ggosiang) [inline docs/primehub-end-to-end-tutorial-3.md:97]: `have a the following` ?

File: docs/primehub-end-to-end-tutorial-3.md [hunk 2]
```diff
@@ -95,96 +164,123 @@ def predict(self, X, feature_names = None, meta = None):
...
-    {"data":{"names":["t:0"],"tensor":{"shape":[1,1],"values":[2.065972089767456]}},"meta":{"requestPath":{"model":"infuseai/tensorflow2-prepackaged:screw-classification"}}}
-    ```
-    The `positive` return value represented as `good` screw; and the `negative` return value represented as `bad` screw.
+## Conclusion
 
-    Therefore, the return value `2.065972089767456` indicated that the requested screw image is a good screw!
+In this tutorial, we compared and selected a suitable model using MLflow, customized a Tensorflow 2 Prepackaged Model Server, deployed a model, and tested the model with sample data.
 
-In the next tutorial, we will build a web app for screw defect detection with our deployed model.
+In part 4, we will build add a web interface to our model using Streamlit, another app available through PrimeHub Apps.
```
Comment (ggosiang) [inline docs/primehub-end-to-end-tutorial-3.md:286]: perhaps remove `build`?

Comment (Dave-infuseai) [reply docs/primehub-end-to-end-tutorial-3.md:286]: Removed 'build'

File: docs/primehub-end-to-end-tutorial-3.md (hunk 1 above)
Comment (Dave-infuseai) [reply docs/primehub-end-to-end-tutorial-3.md:97]: Fixed

**Comment by gblpedia** [comment]
In this screenshot, tutorial_deployment_cell.png, the old card style is replaced in coming v3.8. please use new card style instead. Suggest you can use chrome dev tool to make up a fake one from demo.a.

**Pull request merged**